modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-major.adoc

```diff
@@ -0,0 +1,359 @@
+= Anypoint MQ Connector 3.x Migration Guide
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint MQ Connector version 3.0 includes new features, such as simplified configuration, standardized endpoints, and non-blocking functionality.
+
+=== Global Configuration
+
+In the previous version of the MQ Connector, the `config` element included parameters, as shown in this example:
```

Comment:
According to Mysti, all instances of MQ have to be shown as Anypoint MQ except in code.

Reply:
It's a trademark issue. Thanks for flagging. I checked the spreadsheet.